src/node.rs

```diff
@@ -424,6 +424,11 @@ impl Node {
         self.chain().into_iter().flat_map(Chain::elders)
     }
 
+    /// Returns the members in our section.
+    pub fn our_joined_member_nodes(&self) -> impl Iterator<Item = &P2pNode> {
```

Comment:
Minor: keep consistent naming. Either rename this to `our_joined_members` or rename `Chain::our_joined_members` to `Chain::our_joined_member_nodes`.

Reply:
Changed the function purpose to be known_nodes